### DISCUSSION TITLE
Non-Samsung WearOS watches

### DISCUSSION THREAD

**Discussion by PhilTU** [trupici/G-Watch-Wear#11] [Q&A] [answered]
Does G-watch Wear require specific Samsung libraries or features, or will it work on most WearOS watches ?

**Comment by trupici** [comment, marked as answer]
Gwatch wear app should work on any watch runing on WearOS 3 and higher.
There could be problematic a totally different display with much lower or
higher screen resolution or rectangular shape, but it should work. If you
plan to use another watch face using gwatch complication only then the
display also doesn't matter...

Juraj



Dňa st 19. 7. 2023, 8:20 PhilTU ***@***.***> napísal(a):

**Reply by absampson** [reply]
Does it work on a Samsung Gear S3?

**Reply by PhilTU** [reply]
For S3 you want "G-watch" not "G-watch Wear" as S3 is TizenOS not WearOS.
